commands/audit/audit.go

```diff
@@ -139,7 +139,6 @@ func (auditCmd *AuditCommand) Run() (err error) {
 		SetOutputFormat(auditCmd.OutputFormat()).
 		SetPrintExtendedTable(auditCmd.PrintExtendedTable).
 		SetExtraMessages(messages).
-		SetScanType(services.Dependency).
```

Comment:
why are we removing the scanType?
here and on `commands/scan/buildscan.go` ?

Reply:
I added a new attribute to the result `ResultType` as a `CommandType` a new enum in utils that describes the cmd that the consequence belongs to. so we no longer need to use the `services` enum because we have this info in Results